### ISSUE TITLE
Car Clock

### ISSUE THREAD

**Issue by kennyboy55** [Templarian/MaterialDesign#6222]
### Before submitting a new icon request, please confirm the following:

- [X] I have [searched all issues](https://github.com/Templarian/MaterialDesign/issues) to make sure there isn't a request for this icon.
- [X] I have [searched the current library](https://materialdesignicons.com/) to make sure the icon doesn't already exist.
- [X] I understand that [brand icons are no longer being added](http://dev.materialdesignicons.com/roadmap/brand-icons) and I can use [SimpleIcons](https://simpleicons.org/) for my brand icon needs.
- [X] I only intend to request a single icon (or a few near-identical ones) in this issue.

### What would you use this icon for?

Icon to show travel time by car, similar to the **bus-clock** and **airplane-clock** icon. There are already many _x-clock_ icons, but there are none for certain travel types.

Usage for me is with the Home Assistant integration 'Google Maps Travel Time'. As part of this it would also be great to have similar icons for bicycle, train, walking, tram and subway (I will open new issues if needed).

### Give us an example of what this icon might look like.

Examples of current travel time icons:
![airplane-clock](https://user-images.githubusercontent.com/9005391/136171563-b4c850db-977c-44c4-ba89-eaa082c02f64.png)
![bus-clock](https://user-images.githubusercontent.com/9005391/136171733-489f7664-d2c7-4a85-b982-8894d08759b7.png)

Basically a clock added to this icon:
![car](https://user-images.githubusercontent.com/9005391/136171780-598ce968-5715-4359-87c4-9a4aed9f7146.png)

**Comment by goyney** [comment]
[![car-clock](https://materialdesignicons.com/icon/car-clock)](https://materialdesignicons.com/icon/car-clock)
